src/styles/blocks/calendar.scss

```
@@ -0,0 +1,25 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  max-width: ($cell-size * 7) + 10px;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $cell-size;
+    height: $cell-size;
+    border: $border-width solid black;
+    font-size: $font-size;
+    background-color: #eee;
+    cursor: pointer;
+    transition: background-color $animation-duration;
```

```suggestion
    transition: background-color $animation-duration, transform  $animation-duration;
```
You can do transform transition smooth too